lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -293,6 +293,7 @@ around TO_JSON => sub {
     return {
         %{$self->$orig},
         biography               => $self->biography,
+        birth_date              => datetime_to_iso8601($self->birth_date),
```

Comment:
If we just do `$self->birth_date->TO_JSON` here, we'll get a `{year, month, day}` object, so we won't have to parse anything in the JS.